src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -1728,8 +1730,10 @@ void GrpcLb::OnFallbackTimerLocked() {
 ChannelArgs GrpcLb::CreateChildPolicyArgsLocked(
     bool is_backend_from_grpclb_load_balancer) {
   ChannelArgs r =
-      args_.Set(GRPC_ARG_ADDRESS_IS_BACKEND_FROM_GRPCLB_LOAD_BALANCER,
-                is_backend_from_grpclb_load_balancer);
+      args_
+          .Set(GRPC_ARG_ADDRESS_IS_BACKEND_FROM_GRPCLB_LOAD_BALANCER,
+               is_backend_from_grpclb_load_balancer)
+          .Set(GRPC_ARG_GRPCLB_ENABLE_LOAD_REPORTING_FILTER, 1);
```

Comment:
if `is_backend_from_grpclb_load_balancer == false`, i.e. if we're using fallback addresses, is it still correct to enable the load reporting filter?

Reply:
Arguably no, but that's the existing behavior, so I don't want to change it.